README.md

```diff
@@ -59,11 +59,39 @@ pip install -r requirements.txt
 
 We're using [this library](https://aiohttp.readthedocs.io/en/stable/) to improve speed of polling the TTC's API by making requests asynchronous. Installation was... fine in Ubuntu 16.04 and OSX, had some hiccoughs in Debian/Raspbian. Stay tuned.
 
-### Database setup
+## The Scraper
```

Comment:
I feel like this section can be merged into the above section so there would be (these headers are for descriptive purposes):

## Scraper Setup
### Install Python Things

### Where Data Goes
#### S3 Setup

#### PostgreSQL setup

## Automating Scraper Setup

I'm happy to... do a PR of your PR? Is that a thing? Too much of a headache?